composer.json

```diff
@@ -60,77 +60,82 @@
     }
   ],
   "require": {
-    "consolidation/robo": "~1.2.4",
     "acquia/blt": "9.x-dev",
-    "cweagans/composer-patches": "^1.6",
+    "beleneglorion/php-bugherd-api": "dev-master",
     "composer/installers": "^1.4",
+    "consolidation/robo": "~1.2.4",
+    "cweagans/composer-patches": "^1.6",
     "davidbarratt/custom-installer": "~1.0",
```

Comment:
It would be good to add the prestissimo package here too. https://github.com/hirak/prestissimo

Reply:
Wat does that doe?

Reply:
Cant that be global? thats what they say on that repo

Reply:
So i added that globally, and it really changes up the composer.json file. and makes it really difficult to clear up merge conflicts. I'm not a fan.